packages/components/src/components/Snackbar/Snackbar.stories.tsx

```diff
@@ -34,27 +35,48 @@ export default meta
 
 type Story = StoryObj<SnackbarProps>
 
+const isWeb = Platform.OS !== 'web'
+
 const mobileComponentRenderer = (props: SnackbarProps) => {
-  const { isError, messageA11y, onActionPressed } = props.data || {}
+  const { isError, messageA11y, onActionPressed, offset } = props.data || {}
+  const { show } = useSnackbar()
   const onPressSnackbar = () => {
-    ShowSnackbar(props.message as string, {
+    show(props.message as string, {
       isError,
       messageA11y,
       onActionPressed,
+      offset,
     })
   }
 
-  return <Button label="Press for Snackbar" onPress={onPressSnackbar} />
+  return (
+    <>
+      <Button label="Press for Snackbar" onPress={onPressSnackbar} />
+    </>
+  )
+}
+
+export const _Default: Story = {
```

Comment:
Might be worth checking with Jessica if we wanted this named Default or something else as the only one that'll appear in Storybook.

Reply:
Ended up removing the Custom Offset story as I couldn't get conditional naming of the story to work. I figure it'd be pretty easy for the user to add offset in the args